presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/shared/Standard.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.product.launcher.env.shared;
+
+import io.prestosql.tests.product.launcher.PathResolver;
+import io.prestosql.tests.product.launcher.docker.DockerFiles;
+import io.prestosql.tests.product.launcher.env.DockerContainer;
+import io.prestosql.tests.product.launcher.env.Environment;
+import io.prestosql.tests.product.launcher.env.EnvironmentOptions;
+import org.testcontainers.containers.startupcheck.IsRunningStartupCheckStrategy;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.containers.wait.strategy.WaitAllStrategy;
+
+import javax.inject.Inject;
+
+import java.io.File;
+import java.time.Duration;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static io.prestosql.tests.product.launcher.testcontainers.TestcontainersUtil.exposePort;
+import static java.util.Objects.requireNonNull;
+import static org.testcontainers.containers.BindMode.READ_ONLY;
+
+public final class Standard
```

Comment:
Presto?

Reply:
first, it's presto + tests, not presto alone
second, it's named after standard.yml this corresponds to.
third, please do not rename that file
fourth, we can rename/refactor later, once we forget we had those yml files.